### DISCUSSION TITLE
Filter  / Column of Dataset

### DISCUSSION THREAD

**Discussion by xavier-GitHub76** [apache/superset#28263] [General]
Hello,

## Description

with Superset 3.1.1, it is the physical columns of the table that are listed when defining a filter.

This could be the columns of the dataset, the value displayed should then be the associated label which could have been personalized by the user.

Additionally the calculated columns could also be listed there.

## Screenshot
![Filtre - colonne physique BDD](https://github.com/apache/superset/assets/82046143/983595ff-1427-4c0a-9289-3b1392ef5d52)

Best regards

**Comment by michael-s-molina** [comment]
Moving this to a discussion:

> 💡 Requesting a new feature: Search [GitHub Discussions](https://github.com/apache/superset/discussions) to see if it exists already. If not, add a new post there under "Ideas".
